CHANGELOG.md

```diff
@@ -260,6 +260,11 @@ Other minor additions
   cartesianProduct     : List A → List B → List (A × B)
   ```
 
+* Added new proofs to `Data.List.Properties`:
+  ```agda
+  reverse-injective : ∀ {xs ys : List A} → List.reverse xs ≡ List.reverse ys → xs ≡ ys
```

Comment:
We usually drop all the implicit arguments in the CHANGELOG
(and we can also drop the `List.` qualifier given it's in
`Data.List.Properties`) to keep it light.